webview/src/shared/components/errorTooltip/ErrorTooltip.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ReactElement } from 'react'
 import styles from './styles.module.scss'
-import { Error } from '../../../shared/components/icons'
-import Tooltip from '../../../shared/components/tooltip/Tooltip'
+import { Error } from '../icons'
+import Tooltip from '../tooltip/Tooltip'
```

Comment:
[Q] Should this component be grouped under `tooltip`?

Reply:
I think that would be a good idea. For discoverability mainly.

Reply:
I'll move in a follow up